projects/finder-frontend/docker-compose.yml

```diff
@@ -32,7 +32,7 @@ services:
       - nginx-proxy
     environment:
       ASSET_HOST: finder-frontend.dev.gov.uk
-      VIRTUAL_HOST: finder-frontend.dev.gov.uk
+      VIRTUAL_HOST: http://finder-frontend.dev.gov.uk
```

Comment:
Changing this seems like a bit of a smell where this environment variable is being used beyond it's intention and probably indicates that finder-frontend should not be using it.

This env var is for [nginx-proxy docker container](https://github.com/nginx-proxy/nginx-proxy#docker-compose) and shouldn't be something the application is aware of. 

Hopefully you can use Rails request context to not need to know about the hostname and allow it to redirect based on the incoming request using same hostname and protocol.

Reply:
Thanks Kevin.  I changed my mind immediately after creating the PR (hence why I closed it) and will solve the problem I'm having in another way.